contracts/JBveBanny.sol

```diff
@@ -262,6 +250,13 @@ contract JBveBanny is veERC721, Ownable, ReentrancyGuard, JBOperatable {
     // Mint the position for the beneficiary.
     _safeMint(_beneficiary, tokenId);
 
+    // Enable the voting power if the user is minting for themselves
+    // otherwise the `_beneficiary` has to enable it manually afterwards
+    // TODO: make optional to allow the user to save some gas?
```

Comment:
@xBA5ED why make it optional wouldn't every user locks in would want voting power to get activated on mint ?

Reply:
@xBA5ED also I see the withdraw function is commented in the veNFT Contract have you not got to that part ?

Reply:
same like the 1st point adding here shouldn't redeem also call withdraw in veNFT ?

Reply:
> @xBA5ED why make it optional wouldn't every user locks in would want voting power to get activated on mint ?

Good question, because the gas cost to calculate an users voting power increases with every NFT they have, a possible attack is to send 100 NFTs with 1 token locked to another user. That way calculating their voting power uses more gas than the block limit. Which makes on-chain voting impossible and could also cause off-chain voting to error. 

The fix I thought of is to disable the voting power of an NFT if it's transferred (or minted) to another user. The user then has to enable the voting power manually.

Reply:
> > @xBA5ED why make it optional wouldn't every user locks in would want voting power to get activated on mint ?
> 
> Good question, because the gas cost to calculate an users voting power increases with every NFT they have, a possible attack is to send 100 NFTs with 1 token locked to another user. That way calculating their voting power uses more gas than the block limit. Which makes on-chain voting impossible and could also cause off-chain voting to error.
> 
> The fix I thought of is to disable the voting power of an NFT if it's transferred (or minted) to another user. The user then has to enable the voting power manually.

got it yeah sounds better thanks for the clarity my bad

Reply:
> > same like the 2nd point adding here shouldn't redeem also call withdraw in veNFT ?
> 
> Right now I moved the withdraw logic to the `_burn` method, but im not entirely sure yet if that is the most logical place.

oh I missed that thanks